--- old
+++ new
@@ -1,0 +1,1 @@
+SCHOPENHAUER'S 38 STRATAGEMS, OR 38 WAYS TO WIN AN ARGUMENT
@@ -2,0 +3,91 @@
+Arthur Schopenhauer (1788-1860), was a brilliant German philosopher. These 38 Stratagems are excerpts from "The Art of Controversy", first translated into English and published in 1896.
+
+Schopenhauer's 38 ways to win an argument are:
+
+1. Carry your opponent's proposition beyond its natural limits; exaggerate it. The more general your opponent's statement becomes, the more objections you can find against it. The more restricted and narrow his or her propositions remain, the easier they are to defend by him or her.
+
+2. Use different meanings of your opponent's words to refute his or her argument.
+
+3. Ignore your opponent's proposition, which was intended to refer to a particular thing. Rather, understand
+
+it in some quite different sense, and then refute it. Attack something different than that which was
+
+asserted. 4. Hide your conclusion from your opponent till the end. Mingle your premises here and there in your talk Get your opponent to agree to them in no definite order. By this circuitious route you conceal your game until you have obtained all the admissions that are necessary to reach your goal.
+
+5. Use your opponent's beliefs against him. If the opponent refuses to accept your premises, use his own premises to your advantage.
+
+6. Another plan is to confuse the issue by changing your opponent's words or what he or she seeks to prove.
+
+7. State your proposition and show the truth of it by asking the opponent many questions. By asking many wide-reaching questions at once, you may hide what you want to get admitted. Then you quickly. propound the argument resulting from the opponent's admissions.
+
+8. Make your opponent angry. An angry person is less capable of using judgement or perceiving where his or her advantage lies.
+
+9. Use your opponent's answers to your questions to reach different or even opposite conclusions.
+
+10. If your opponent answers all your questions negatively and refuses to grant any points, ask him or her to concede the opposite of your premises. This may confuse the opponent as to which point you actually seek them to concede.
+
+11. If the opponent grants you the truth of some of your premises, refrain from asking him or her to agree to your conclusion. Later, introduce your conclusion as a settled and admitted fact. Your opponent may. come to believe that your conclusion was admitted.
+
+12. If the argument turns upon general ideas with no particular names, you must use language or a metaphor that is favorable in your proposition.
+
+13. To make your opponent accept a proposition, you must give him or her an opposite, counter-proposition as well. If the contrast is glaring, the opponent will accept your proposition to avoid being paradoxical.
+
+14. Try to bluff your opponent. If he or she has answered several of your questions without the answers
+
+turning out in favor of your conclusion, advance your conclusion triumphantly, even if it does not follow
+
+If your opponent is shy or stupid, and you yourself possess a great deal of impudence and a good voice, the trick may easily succeed. 15. If you wish to advance a proposition that is difficult to prove, put it aside for the moment. Instead, submit
+
+for your opponent's acceptance or rejection some true poposition, as thoug you wished to draw your proof from it. Should the opponent reject it because he or she suspects a trick, you can obtain your triumph by showing how absurd the opponent is to reject a true proposition. Should the opponent accept it, you now have reason on your own for the moment. You can either try to prove your original proposition or maintain that your original proposition is proved by what the opponent accepted. For this,
+
+an extreme degree of impudence is required. 16. When your opponent puts forth a proposition, find it inconsistent with his or her other statements,
+
+beliefs, actions, or lack of action.
+
+17. If your opponent presses you with a counter proof, you will often be able to save yourself by advancing
+
+some subtle distinction. Try to find a second meaning or an ambiguous sense for your opponent's idea. 18. If your opponent has taken up a line of argument that will end in your defeat, you must not allow him or her to carry it to its conclusion. Interrupt the dispute, break it off altogether, or lead the opponent to a
+
+different subject. 19. Should your opponent expressly challenge you to produce any objection to some definite point in his or her argument, and you have nothing much to say, try to make the argument less specific.
+
+20. If your opponent has admitted to all or most of your premises, do not ask him or her directly to accept your conclusion. Rather draw the conclusion yourself as if it too had been admitted. 21. When your opponent uses an argument that is superficial, refute it by setting forth its superficial
+
+character. But it is better to meet the opponent with a counter argument that is just as superficial, and so dispose of him or her. For it is with victory that your are concerned, and not with truth. 22. If your opponent asks you to admit something from which the point in dispute will immediately follow,
+
+you must refuse to do so, declaring that it begs the question. 23. Contradiction and contention irritate a person into exaggerating his or her statements. By contractiong your opponent you may drive him or her into extending the statement beyond its natural limit. When you then contradict the exaggerated form of it, you look as though you had refuted the orginal statement your opponent tries to extend your own statement further than you intended, redefine your statement's limits. 24. This trick consists in stating a false syllogism. Your opponent makes a proposition and by false inference and distortion of his or her ideas you force from the proposition other propositions that are not intended.
+
+and that appear absurd. It then appears the opponent's proposition gave rise to these inconsistencies, and
+
+so appears to be indirectly refuted.
+
+25. If your opponent is making a generalization, find an instance to the contrary. Only one valid contradiciton is needed to overthrow the opponent's proposition. 26. A brilliant move is to turn the tables and use your opponent's arguments against him or herself.
+
+27. Should your opponent surprise you by becoming particularly angry at an argument, you must urge it with all the more zeal. Not only will this make the opponent angry, it may be presumed that you put your finger on the weak side of his or her case, and that the opponent is more open to attack on this point
+
+than you expected. 28. This trick is chiefly practicable in a dispute if there is an audience who is not an expert on the subject. You make an invalid objection to your opponent who seems to be defeated in the eyes of the audience. This strategy is particularly effective if your objection makes the opponent look ridiculous or if the audience laughs. If the opponent must make a long, complicated explanation to correct you, the audience will not be disposed to listen.
+
+29. If you find that you are being beaten, you can create a diversion that is, you can suddenly begin to talk of something else, as though it had bearing on the matter in dispose. This may be done without presumption. if the diversion has some general bearing on the matter.
+
+30. Make an appeal to authority rather than reason. If your opponent respects an authority or an expert, quote that authority to further your case. If needed, quote what the authority said in some other sense or circumstance. Authorities that your opponent fails to understand are those which he or she generally
+
+admires the most. You may also, should it be necessary, not only twist your authorities, but actually falsify them, or quote something that you have invented entirely yourself. 31. If you know that you have no reply to an argument that your opponent advances, you may, by a fine
+
+stroke of irony, declare yourself to be an incompetent judge. 32. A quick way of getting rid of an opponent's assertion, or throwing suspicion on it, is by putting it into some odious category.
+
+33. You admit your opponent's premises but deny the conclusion. 34. When you state a question or an argument, and your opponent gives you no direct answer, or evades it with a counter question, or tries to change the subject, it is a sure sign you have touched a weak spot, sometimes without knowing it. You have as it were, reduced the opponent to silence. You must, therefore,
+
+urge the point all the more, and not let your opponent evade it, even when you do not know where the weakness that you have hit upon really lies. 35. This trick makes all unnecessary if it works. Instead of working on an opponent's intellect, work on his or
+
+her motive. If you succeed in making your opponent's opinion, should it prove true, seem distinctly to his or her own interest, the opponenent will drop it like a hot potato.
+
+36. You may also puzzle and bewilder your opponent by mere bombast. If the opponent is weak or does not wish to appear as ife he or she has no idea what you are talking about, you can easily impose upon him or her some argument that sounds very deep or learned, or that sounds indisputable.
+
+37. Should your opponent be in the right but, luckily for you, choose a faulty proof, you can easily refute it and then claim that you have refuted the whole position. This is the way which bad advocates lose a good case. If no accurate proof occurs to the opponent or the bystanders, you have won the day. 38. A last trick is to become personal, insulting and rude as soon as you perceive that your opponent has the
+
+upper hand. In becoming personal you leave the subject altogether, and turn your attack on the person by
+
+remarks of an offensive and spiteful character. This is a very popular trick, because everyone is able to carry it into effect.
+
+Boswell and Dan Starer)
+
+(abstracted from the book:Numerical Lists You Never Knew or Once Knew and Probably Forget, by: John